content/en/build/storage-helpers.md

```diff
@@ -23,7 +23,7 @@ Storage helpers are available for NFTs (non-fungible tokens) or general data. If
 
 - [Estuary](https://docs.estuary.tech) is a free, decentralized data storage service for IPFS and Filecoin. Users can store and retrieve content quickly, and have their data backed up with proposition receipts and successful deal receipts. Estuary’s API adheres to the IPFS Pinning Services API Spec. You can use Estuary via its HTTP API or rclone syncer. Note: Estuary is in alpha mode. Users wanting to store meaningful public data can apply for an API token.
 
-- [Web3.Storage](https://web3.storage/docs) is a fast, open, and free service for decentralized web storage. Upload any data, and it will end up on a decentralized set of IPFS and Filecoin storage providers. There are JavaScript and Go libraries for the API, as well as a web uploader.
+- [Web3.Storage](https://web3.storage/docs) is a fast and open developer platform for storing and interacting with data in a decentralized way. Upload any data, and it will end up on a decentralized set of IPFS and Filecoin storage providers. There are JavaScript and Go libraries for the API, as well as a web uploader. You can read more about the vision of the web3.storage platform [here](https://blog.web3.storage/posts/say-hello-to-the-data-layer-1-3-intro-to-web3-storage).
```

Comment:
```suggestion
- [Web3.Storage](https://web3.storage/docs) is a fast and open developer platform for [ storing and interacting with data](https://blog.web3.storage/posts/say-hello-to-the-data-layer-1-3-intro-to-web3-storage). Upload any data, and Web3.storage will ensure it ends up on a decentralized set of IPFS and Filecoin storage providers. There are JavaScript and Go libraries for the API, as well as a no-code web uploader. Free and paid plans are available.
```

- used active voice (web3.storage will ensure...) instead of passive (data will end up)
- for SEO and legibility, linktext should describe the content you'll get, rather than generic "here"
- what do you think about explicitly saying "free and paid plans are available"?